### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by pydctw** [kubernetes-sigs/cluster-api-provider-gcp#583] [Ideas]
What features do you want to see in CAPG? What should we work on to make CAPG production ready?

Please add your ideas and share links to issues below. You can upvote issues that you would like to see in upcoming releases. 

Notes
* Let's keep the discussion thread to high-level ideas and issues only, and discuss each issue  in the issue's comment.
* Features requiring breaking API changes will need to wait for the next API version (v1beta2). You are welcome to add the feature.

**Comment by pydctw** [comment]
Support GKE based clusters
* https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/512

**Comment by pydctw** [comment]
GCP Machine Pool support using Managed Instance Groups
* https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/297

**Comment by pydctw** [comment]
Doc - improve doc and publish them
* https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/490

**Comment by pydctw** [comment]
Implement Conditions
* https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/570

**Comment by pydctw** [comment]
Support multi-tenancy
* https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/575

**Comment by aniruddha2000** [comment]
GPU support

- [GPU support #289](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/289)
